src/doc/rustc-dev-guide/src/tests/running.md

```diff
@@ -240,28 +240,6 @@ and they may not run the same without those options.
 
 ## Running `run-make` tests
```

Comment:
This whole section can now be removed?

Reply:
Yes. Somehow I left this title behind with no content lol.